src/components/vega-lite/index.tsx

```diff
@@ -16,6 +16,8 @@ export interface VegaLiteProps {
   logger: Logger;
 
   data: InlineData;
+
+  viewListener?: (view: vega.View) => any;
```

Comment:
Are there other possible even that can be listen in Vega? 

If so, this could be viewRunAfter / viewRunAfterListener or something just in case we add other view listeners in the future?